ofl/mooli/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,3 @@
+<p>Mooli is a Sans Serif font designed for young readers. Mooli is derived from the <a href="https://fonts.google.com/specimen/Mulish">Muli</a> font family. It has been designed to be used freely across the internet by web browsers on desktop computers, laptops and mobile devices.</p>
```

Comment:
"Mulish" instead of "Muli" (the name was changed because it was offensive in some languages)
Would be good to have few words about what are the difference with the original Mulish